readme.md

```diff
@@ -17,15 +17,15 @@ The following roadmap gives an idea of the various packages that compose the `go
 ```bash
 ├── abi ✅                     # types for abi encoding and decoding.
 ├── channel ✅                 # query the latest supported state of a channel
-│   ├── consensus_channel 🚧   # manage a running ledger channel.
+│   ├── consensus_channel ✅    # manage a running ledger channel.
 │   └── state ✅               # generate and recover signatures on state updates
 │       ├── outcome ✅         # define how funds are dispersed when a channel closes
-├── client 🚧                  # exposes an API to the consuming application
-│   └── engine 🚧              # coordinate the client components, runs the protocols
-│       ├── chainservice 🚧    # watch the chain and submit transactions
-│       ├── messageservice 🚧  # send and recieves messages from peers
-│       └── store 🚧           # store keys, state updates and other critical data
-├── client_test 🚧             # integration tests involving multiple clients
+├── client ✅                  # exposes an API to the consuming application
+│   └── engine ✅              # coordinate the client components, runs the protocols
+│       ├── chainservice ✅   # watch the chain and submit transactions
```

Comment:
I wasn't sure if these should be marked as complete or not since we just have a mock-chain and a memory store.

Reply:
I think we should leave them as "in progress" until we have the "proper" services built.